openjdk/excludes/vendors/microsoft/ProblemList_openjdk17.txt

```diff
@@ -99,6 +99,7 @@ java/lang/module/ModuleDescriptorHashCodeTest.java https://bugs.openjdk.org/brow
 tools/jlink/plugins/VendorInfoPluginsTest.java https://github.com/adoptium/aqa-tests/issues/4155 windows-aarch64
 sun/tools/jhsdb/HeapDumpTest.java https://github.com/adoptium/aqa-tests/issues/4155 windows-aarch64
 sun/tools/jhsdb/JShellHeapDumpTest.java https://github.com/adoptium/aqa-tests/issues/4155 windows-all
+sun/tools/jhsdb/JStackStressTest.java https://github.com/adoptium/aqa-tests/issues/5499 windows-x64
```

Comment:
IIRC we usually reference the JBS issue, not sure of the syntax

Reply:
Previously, we have used Github issues. If this is not correct, please let me know! To add, I did not find an exact JBS bug for this issue only the related one linked in the description. We do get a similar, or exactly the same, error as described in the JBS bug.